src/megatron/energon/av/decode_av_frames.py

```diff
@@ -47,7 +47,7 @@ def get_frames(
         video_decode_audio: bool = False,
         video_num_frames: int = 64,
         video_out_frame_size: tuple[int, int] = (224, 224),
-    ) -> VideoData | None:
+    ) -> Optional[VideoData]:
```

Comment:
nit: `Optional` doesn't imply that `None` will always be returned if `VideoData` is not, so this is actually less clear.

Reply:
Resolved: `Optional` and `| None` are equivalent: https://docs.python.org/3.10/library/typing.html#typing.Optional

Reply:
resolved in offline discussion: VideoData | None needs python 3.10 and Optional is functionally equivalent